Apps/Src/common.c

```diff
@@ -9,3 +9,11 @@ void print_float(char * str, float f) {
     int32_t d = (f<0)?-f:f;
     printf("%d.%02d\n\r", (int)n, (int)d);
 }
+
+void print_bin(char * str, uint32_t i){
+    if(str) printf(str);
+
+    for(uint32_t mask=0x80000000L; mask > 0L; mask >>= 1){
+        printf("%d",(mask & i)?1:0);
+    }
```

Comment:
Should this function print a newline, to match the float printing function?

Reply:
I think that makes sense, so I'll add \n\r, but @IshDeshpa let me know if that was intentional and I'll change it back to how it is